api/src/main/java/javax/enterprise/inject/Instance.java

```diff
@@ -203,5 +203,17 @@
      *         destroying bean instances
      */
     public void destroy(T instance);
+    
+    /**
+     * <p>
+     * Instructs the container to destroy the instance. The instance passed should be a dependent scoped bean instance
```

Comment:
```should be``` sounds weird here.
I suppose that if it _isn't_ a dependent scoped bean, you just ignore it.
If that is the case, I would rather say ```... instance passed has to be a dependent ...```.